scipy/spatial/distance.py

```diff
@@ -2068,6 +2115,13 @@ def cdist(XA, XB, metric='euclidean', p=2, V=None, VI=None, w=None):
 
          dm = cdist(XA, XB, 'sokalsneath')
 
+    24. ``Y = cdist(XA, XB, 'aitchison')``
+
+       Computes the distance between :math:`m` points using
+       Aitchison distance (2-norm) as the distance metric between the
```

Comment:
This is confusing, it's not just the 2-norm right? If it is, then isn't it equal to ``minkowski(u, v, 2)``?